include/unified_teleop/pnt_stmp_modifiers.hpp

```
@@ -0,0 +1,217 @@
+    /// @return A modified PointStamped message with adjusted values for the specified axis.
+    geometry_msgs::msg::PointStamped adjust_mirror_joy(const std::optional<rosnu::MovementInput> input,
+                                                        const geometry_msgs::msg::PointStamped orig_message,
+                                                        const AxisType axis_type,
+                                                        const bool is_increasing,
+                                                        const sensor_msgs::msg::Joy joy_state)
+    {
+        // If the provided input is null (not initialized), return the original message without modifications
+        if (!input)
+        {
+            return orig_message;
+        }
+        
+        // Create a new PointStamped message starting as a copy of the original message
+        geometry_msgs::msg::PointStamped new_message = orig_message;
+        double reading = 0.0f; // Initialize a variable to store the controller input's value
+        
+        // Determine the type of the input (Axis, Trigger, Button) and read its value from the joy_state
+        switch (input->type)
+        {
+            case rosnu::InputType::Axis:
+                // For an axis input, directly take its reading
+                reading = joy_state.axes.at(input->index);
+                break;
+            case rosnu::InputType::Trigger:
+                // For a trigger, convert its value (normally -1.0 to 1.0) to a range of 0.0 to 1.0
+                reading = 0.5 - (joy_state.axes.at(input->index)/2.0);
+                break;
+            case rosnu::InputType::Button:
+                // For a button, take its binary state (pressed or not pressed)
+                reading = joy_state.buttons.at(input->index);
+                break;
+            case rosnu::InputType::None:
+                // Log an error and exit if the input type is uninitialized
```

Should throw an exception here.
Need to handle the default case